docs/_assets/attributes.adoc

```diff
@@ -70,7 +70,7 @@
 :unstable-api-version: denoted by a major version 0
 :stable-api-version: version 1.0.0
 
-:DocRoot: ..
+ifndef::DocRoot[:DocRoot: ..]
```

Comment:
Might be worth commenting to help us remember why the workaround is required.
@PaulRMellor FYI